var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -166,6 +169,12 @@ def setup_run_environment(self, env):
                 os.path.join(self.spec.prefix, self.lua_lib_dir, '?.so'),
                 separator=';')
 
+    @run_after('install')
+    def link_pkg_config(self):
+        if "+pcfile" in self.spec:
+            symlink(join_path(self.prefix.lib, 'pkgconfig', 'lua-5.3.pc'),
+                    join_path(self.prefix.lib, 'pkgconfig', 'lua.pc'))
```

Comment:
This has a dash too much on my platform (Ubuntu 18.04):
![Screenshot from 2021-03-22 11-25-34](https://user-images.githubusercontent.com/4199709/111975985-750dd100-8b01-11eb-9cc5-8d37b51d6751.png)

I think **this** is what breaks the build, not retrieving the patch from remote. Can you confirm?

Reply:
@v I can build `rpm` with:

[rpm.patch.txt](https://github.com/spack/spack/files/6181706/rpm.patch.txt)

and deleting the patch from the repository. The issue I had with `gettext` [is a FAQ](https://www.gnu.org/software/gettext/FAQ.html#integrating_undefined), maybe we should follow with another PR and check if this needs to be solved in the gettext package.

Reply:
I also got that error at some point - I could follow up with a PR, and what would you suggest I change for gettext?